views/book_form.pug

```diff
@@ -3,31 +3,38 @@ extends layout
 block content
   h1= title
 
-  form(method='POST' action='')
+  form(method='POST')
     div.form-group
       label(for='title') Title:
-      input#title.form-control(type='text', placeholder='Name of book' name='title' required='true' value=(undefined===book ? '' : book.title) )
+      input#title.form-control(type='text', placeholder='Name of book' name='title' required value=(undefined===book ? '' : book.title) )
     div.form-group
       label(for='author') Author:
-      select#author.form-control(type='select', placeholder='Select author' name='author' required='true' )
+      select#author.form-control(name='author' required)
+        option(value='') --Please select an author--
         for author in authors
           if book
-            option(value=author._id selected=(author._id.toString()===book.author._id.toString() ? 'selected' : false) ) #{author.name}
+            if author._id.toString()===book.author._id.toString()
+              option(value=author._id selected) #{author.name}
+            else
+              option(value=author._id) #{author.name}
           else
             option(value=author._id) #{author.name}
     div.form-group
       label(for='summary') Summary:
-      input#summary.form-control(type='textarea', placeholder='Summary' name='summary' value=(undefined===book ? '' : book.summary) required='true')
+      textarea#summary.form-control(placeholder='Summary' name='summary' required)= undefined===book ? '' : book.summary
```

Comment:
Hmmm. I'm OK with this, but using an input is just as valid as using a textarea element.

Reply:
The problem is that <input type='textarea'> is invalid, because this [input type](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types) does not exist. One must use the [textarea](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) element instead.

Reply:
THanks!